webrtc-stats.html

```diff
@@ -2218,6 +2220,40 @@ <h2>
                   formats have ways to vary the quantizer value within the frame.
                 </p>
               </dd>
+              <dt>
+                <dfn>psnrSum</dfn> of type <span class="idlMemberType">
+                record&lt;DOMString, double&gt;</span>
+              </dt>
+              <dd>
+                <p>
+                  MUST NOT [= map/exist =] for audio.
+                  The cumulative sum of the PSNR values of frames encoded by this sender.
+                  The record includes values for the "y", "u" and "v" components.
+                  The count of measurements is in {{psnrMeasurements}}.
+                </p>
+                <p>
+                  PSNR is defined in [[ISO-29170-1:2017]].
+                </p>
+                <p class="note">
+		  PSNR metrics should primarily be used as a basis for statistical analysis rather
```

Comment:
Avoid "should" in non-normative notes. How about
```suggestion
		  Authors are expected to use PSNR metrics primarily as a basis for statistical analysis rather
```

Reply:
https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-totalcorruptionprobability -- literally copied from this note, can you explain why it is ok there but not here?

Reply:
I missed it in review of https://github.com/w3c/webrtc-stats/pull/788. It's not a [hard rule](https://w3c.github.io/webrtc-stats/#conformance), but most WG editors I've spoken with agree avoiding lowercase requirement-laden words avoids confusion and improves readability.

Both notes appear to be speaking to [authors](https://w3ctag.github.io/design-principles/#priority-of-constituencies) rather than implementers, which is fine. But the primary audience of specs are implementers, so it might help to clarify when speaking to someone else. Specs have no authority saying authors should or shouldn't do anything. So it seems more accurate to describe the usage the design anticipates, which is how I interpret these notes.

Reply:
No, this note documents that implementers (well, the one) do not think authors should be abusing this for per-frame analysis. 810d67d6b4894aba56c8404380baa497dbfb2110 avoids lower-case should.